### PR TITLE
ZetaSsp Bid Adapter: add new bid adapter

### PR DESCRIPTION
**Pull request by asurovenko-zeta**
## Type of change
- [ ] New bidder adapter

## Description of change
- [ ] New bidder adapter

- test parameters for validating bids
```
{
    bidder: 'zeta_global_ssp',
    bidId: 12345,
    params: {
        placement: 12345,
        user: {
            uid: 12345,
            buyeruid: 12345
        },
        tags: {
            someTag: 123
        },
        test: 1
    }
}
```

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@asurovenko-zeta can you please also make sure and submit a docs request to go along with this one? There are decently detailed instructions [here](https://docs.prebid.org/dev-docs/bidder-adaptor.html#required-files) if you scroll to the bottom of the pafe

**Comment by asurovenko-zeta** [comment]
@ChrisHuie docs was added here: https://github.com/prebid/prebid.github.io/pull/2782

File: modules/zetaSspBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      let bid = {
+        requestId: zetaBid.impid,
+        cpm: zetaBid.price,
+        currency: zetaResponse.cur,
+        width: zetaBid.w,
+        height: zetaBid.h,
+        ad: zetaBid.adm,
+        ttl: TTL,
+        creativeId: zetaBid.crid,
+        netRevenue: NET_REV
```
Comment (patmmccann) [inline modules/zetaSspBidAdapter.js:111]: are you able to set meta.advertiserDomains?

Comment (asurovenko-zeta) [reply modules/zetaSspBidAdapter.js]: advertiserDomains was added along with test update

Comment (patmmccann) [reply modules/zetaSspBidAdapter.js]: thank you!

**Comment by ncolletti** [comment]
@asurovenko-zeta Given that the only items changing in this release compared to your current adapter are the endpoint and a new param, we suggest that you close this PR and instead update your existing adapter to handle a new bidder code for 'zeta_global_ssp'. This bidder code when read by your adapter will use the correct endpoint for the request and will handle the new param as well. You can look toward the AOL adapter for inspiration on how they achieved this same setup.

**Comment by asurovenko-zeta** [comment]
@ncolletti we can't resolve which bidder code must be defined for request because all of different request params is optional and there are a lot of situations with almost the same request.

**Comment by ncolletti** [comment]
Sorry I am not following why this is a problem? You would add aliases to your adapter, then look at the bid.bidder to determine how to read the rest of the params sent in the bidRequest. At that point you can point it to the correct endpoint and use the new param you are introducing with this PR in ext.tags

**Comment by asurovenko-zeta** [comment]
@ncolletti
We are planned making other changes in new Ssp Zeta Adapter in the future, because of that we made the desicion that we would like to ask you to approve the current version to facilitate future changes.

CC: miakovlev@zetaglobal.com

**Comment by asurovenko-zeta** [comment]
@ncolletti
Could you please look at this PR? I would like to speed up the process a little, our team is looking forward to this PR.
Thanks a lot!

**Comment by ncolletti** [comment]
OK we can continue with this PR and submission of a separate adapter. It was noticed that both adapters are asking the user to pass geo within your private params, this information is generally considered as first party data and should be made available by the publisher to Prebid via the ortb2 object and setConfig, [details here](https://docs.prebid.org/features/firstPartyData.html#in-page-examples). In order to be more aligned with our FPD initiative, would you be open to updating geo to look in this location (ortb2.device.geo.country) instead of params?

**Comment by asurovenko-zeta** [comment]
@ncolletti
We're happy to stick with your FPD initiative, I hope current solution on it.

**Comment by ncolletti** [comment]
Thank you @asurovenko-zeta - Could you also update your docs pr to reflect no longer accepting device in params?  Eventually it would be great to remove this also from your other adapter as well as to look to the 'ortb2' first party data object in the global prebid context for the rest of these params (site, user, app). Maybe it makes sense in a separate commit to make that change to fall back to that location if they aren't in params.

**Comment by asurovenko-zeta** [comment]
@ncolletti
The PR to docs repo was added: https://github.com/prebid/prebid.github.io/pull/2837
I have no authority to change our DSP adapter, but I'll pass the information to the DSP team.
Thanks!

**Review by ncolletti — APPROVED** [review]
LGTM. Would be great to consider another PR to use prebid 'ortb2' locations for the remaining objects in your params in this adapter. Thanks!

**Pull request merged**